pyproject.toml

```diff
@@ -90,11 +90,11 @@ strict_equality = true
 ignore_missing_imports = true
 
 [tool.cibuildwheel]
-skip = "pp*"
+# Skip musllinux builds for the moment
+skip = "*-musllinux_*"
 test-extras = "test"
 test-command = "pytest {project}/tests"
 manylinux-x86_64-image = "manylinux2014"
-manylinux-i686-image = "manylinux2014"
```

Comment:
Do you need 32-bit linux? We've removed it from `"auto"`, so you need to add `"auto32"` for it. I don't think numpy ships 32-bit linux wheels?

Reply:
We can also now try building the musl linux variant FWIW